### PR TITLE
Removal of the Living Assets Ownership Pallet

### PR DESCRIPTION
**Pull request by ccubu**
## PR Type:
Refactoring

___
## PR Description:
This PR focuses on the removal of the Living Assets Ownership Pallet from the ownership-chain project. The changes include:
- Deletion of the Living Assets Ownership Pallet related files.
- Removal of the related precompiles from the precompiles module.
- Adjustment of the Runtime configuration in the precompiles module.

___
## PR Main Files Walkthrough:
<details> <summary>files:</summary>

- `ownership-chain/runtime/src/precompiles/mod.rs`: The Living Assets Ownership Pallet related precompiles and types have been removed. This includes the removal of the `LivingAssetsPrecompile` and `Erc721` types, and their associated execution in the `execute` function. The `is_precompile` function has also been updated to remove the check for collection addresses.
- `ownership-chain/pallets/living-assets-ownership/src/lib.rs`: The entire file has been deleted as part of the removal of the Living Assets Ownership Pallet.
- `ownership-chain/precompile/living-assets/src/lib.rs`: The entire file has been deleted as it was part of the Living Assets Ownership Pallet precompiles.
- `ownership-chain/precompile/erc721/src/lib.rs`: The entire file has been deleted as it was part of the ERC721 precompiles associated with the Living Assets Ownership Pallet.
- `ownership-chain/pallets/living-assets-ownership/Cargo.toml`: The entire file has been deleted as it was the Cargo.toml file for the Living Assets Ownership Pallet.
</details>

### CODE REVIEW

File: ownership-chain/runtime/src/precompiles/test.rs [hunk 1]
```diff
@@ -23,8 +22,5 @@ fn ethereum_precompiled_addresses_are_precompile() {
 	assert!(is_precompile(hash(3)).unwrap());
 	assert!(is_precompile(hash(4)).unwrap());
 	assert!(is_precompile(hash(5)).unwrap());
-	assert!(is_precompile(hash(1026)).unwrap());
```
Comment (asiniscalchi) [inline ownership-chain/runtime/src/precompiles/test.rs:26]: I have the suspect that this test will not fail...

**Review by asiniscalchi — CHANGES_REQUESTED** [review]

**Comment by asiniscalchi** [comment]
the change request expose a problem in the design. However we won't address it now but fix the behavior.

File: ownership-chain/runtime/src/precompiles/mod.rs [hunk 2]
```diff
@@ -61,18 +49,12 @@ where
 			// Non-Frontier specific nor Ethereum precompiles :
 			// a if a == hash(1024) => Some(Sha3FIPS256::execute(handle)),
 			a if a == hash(1025) => Some(ECRecoverPublicKey::execute(handle)),
-			a if a == hash(1026) => Some(LivingAssetsPrecompile::execute(handle)),
```
Comment (dastanfv) [inline ownership-chain/runtime/src/precompiles/mod.rs:64]: should be removed from `pub fn used_addresses()` as well

Comment (ccubu) [reply ownership-chain/runtime/src/precompiles/mod.rs:64]: done

File: ownership-chain/runtime/src/precompiles/test.rs (hunk 1 above)
Comment (ccubu) [reply ownership-chain/runtime/src/precompiles/test.rs:26]: Added check for hash(1026) not being a precompile

**Review by asiniscalchi — APPROVED** [review]

**Pull request merged**